.idea/runConfigurations/Run_Robot_Tests.xml

```diff
@@ -0,0 +1,23 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Run Robot Tests" type="GradleRunConfiguration" factoryName="Gradle">
+    <ExternalSystemSettings>
+      <option name="executionName" />
+      <option name="externalProjectPath" value="$PROJECT_DIR$" />
+      <option name="externalSystemIdString" value="GRADLE" />
+      <option name="scriptParameters" value="-PFULL_TEST=true -PUSE_ROBOT=true" />
```

Comment:
Is this also possible for manual tests? Or how do you launch manual tests?

Reply:
I do it by clicking on the run icon of the main method. Manual tests uses non-standard folder structure, so I had to customize the gradle script. As they are not part of the build, it's not possible (I think) to create run configurations.